api/v1/services/RSVPService.js

```diff
@@ -22,6 +25,18 @@ module.exports.getRSVPById = (id) => RSVP.findById(id);
  * @throws {InvalidParameterError} thrown when an attendee already has an rsvp
  */
 module.exports.createRSVP = (attendee, user, attributes) => {
+
+  StatsService.incrementStat('rsvp', 'school', attendee.get('school'));
```

Comment:
preferably the controller has a const array which designates which fields to increment and then you can call the stat update function for-each element of the array